tests/test_regression.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import polars as pl
+import mass2
+import pulsedata
+
+
+def load_data():
```

Comment:
Wow, you're mixing and matching ljh files from different detectors, start dates, runs, etc. Is this a feature or an anti-feature? If I were to look at `data.header.description` I would have no idea you had done that. Maybe `data.with_more_channels` should update the `header.description` to indicate it's a franken-`Channels`?

Reply:
It's better to have potentially wrong "watch out this is wrong" message, than a potentiall wrong "all good, proceed" message. I'd update it to either be `data1.header.description+data2.header.description` or "results of `Channels.add_more_channels`